MekHQ/src/mekhq/gui/model/UnitTableModel.java

```diff
@@ -125,16 +126,80 @@ public int getAlignment(int col) {
-        return switch (col) {
-            case COL_STATUS -> u.isRefitting() ? u.getRefit().getDesc() : null;
-            case COL_CREW_STATE -> u.getCrewState().getToolTipText();
-            case COL_QUIRKS -> u.getQuirksList();
+    /**
+     * Returns the tooltip for the specified row and column.
+     *
+     * @param rowIndex    the index of the row
+     * @param columnIndex the index of the column
+     * @return the tooltip for the specified row and column, or {@code null} if no tooltip is available
+     */
+    public @Nullable String getTooltip(int rowIndex, int columnIndex) {
+        Unit unit = getUnit(rowIndex);
+
+        return switch (columnIndex) {
+            case COL_STATUS -> unit.isRefitting() ? unit.getRefit().getDesc() : null;
+            case COL_CREW_STATE -> unit.getCrewState().getToolTipText();
+            case COL_CREW -> getCrewTooltip(unit);
+            case COL_QUIRKS -> unit.getQuirksList();
             default -> null;
         };
     }
 
+    /**
+     * Returns the tooltip for the crew status of a given unit.
+     *
+     * @param unit the unit for which to get the crew tooltip
+     * @return the crew tooltip as an HTML string
+     */
+    static String getCrewTooltip(Unit unit) {
+        int gunnersNeeded = unit.getTotalGunnerNeeds();
+        int gunnersAssigned = unit.getGunners().size();
+
+        int driversNeeded = unit.getTotalDriverNeeds();
+        int driversAssigned = unit.getDrivers().size();
+
+        Entity entity = unit.getEntity();
+        int navigatorsNeeded = entity instanceof Jumpship && !(entity instanceof SpaceStation) ? 1 : 0;
+        int navigatorsAssigned = unit.getNavigator() == null ? 0 : 1;
+
+        int crewNeeded = unit.getTotalCrewNeeds();
+        int crewAssigned = unit.getCrew().size() - (gunnersAssigned + driversAssigned + navigatorsAssigned);
+
+        StringBuilder report = new StringBuilder("<html>");
+
+        if (driversNeeded > 0) {
+            appendReport(report, "Drivers", driversAssigned, driversNeeded);
+        }
+
+        if (gunnersNeeded > 0) {
+            appendReport(report, "Gunners", gunnersAssigned, gunnersNeeded);
+        }
+
+        if (crewNeeded > 0) {
+            appendReport(report, "Crew", crewAssigned, crewNeeded);
+        }
+
+        if (navigatorsNeeded > 0) {
+            appendReport(report, "Navigator", navigatorsAssigned, navigatorsNeeded);
+        }
+
+        report.append("</html>");
+
+        return report.toString();
+    }
+
+    /**
+     * Appends a crew report line to the provided StringBuilder.
+     *
+     * @param report the {@link StringBuilder} to append to
+     * @param title the title of the crew role (e.g., "Driver", "Gunner")
+     * @param assigned the number of crew members assigned to the role
+     * @param needed the number of crew members needed for the role
+     */
+    static void appendReport(StringBuilder report, String title, int assigned, int needed) {
```

Comment:
also private

Reply:
Unless I'm thinking about things wrongly, this needs to be package-private so it's accessible to the unit test